ckanext/csrf_filter/plugin.py

```diff
@@ -4,13 +4,13 @@
 
 from logging import getLogger
 
-from ckan.plugins import toolkit, SingletonPlugin, implements, \
-    IConfigurable, IRoutes, IBlueprint, IMiddleware
+from ckan import plugins
+from ckan.plugins import implements, toolkit
 
 from ckanext.csrf_filter import anti_csrf
 
 
-if toolkit.check_ckan_version(min_version='2.8.0'):
+if toolkit.check_ckan_version('2.8'):
```

Comment:
so don't need full name?

Reply:
No, it's not necessary. A minimum version check is by far more common than max version check, so they made it the default.